compiler/src/compile.rs

```diff
@@ -975,7 +984,7 @@ impl<O: OutputStream> Compiler<O> {
 
         // Turn code object into function object:
         self.emit(Instruction::MakeFunction {
-            flags: bytecode::FunctionOpArg::empty(),
+            flags: bytecode::FunctionOpArg::NO_NEW_LOCALS,
```

Comment:
Instead of using the flag `NO_NEW_LOCALS`, it might be good to flag the inverse `NEW_LOCALS`. This is in line with CPython, plus it indicates the presence of new locals, instead of the absence of new locals. I find it usually prefer non-inverted booleans. This to prevent constructions like this:

```rust
if !no_new_locals {
  // double negation
  // this block is executed when we have new locals.
}
```

For example, I would turn this code:
```rust
if !x {
  // do a
} else {
  // do b
}
```

into this:

```rust
if x {
  // do b
} else {
  // do a
}

```

Reply:
I did this because it's a very specific special case where we don't want new locals, and every other time we emit a `MakeFunction` instruction we'd have to remember to include the `NEW_LOCALS` flag.

Reply:
Okay, I see. In this case, could be implement the `Default::default` trait to implement this sensible default on the bitflags?